### PR TITLE
Update for current reflex, bump version

### LINKED ISSUE #6: Update to reflex-0.5
Is anyone working on updating this to reflex-0.5 and reflex-dom-0.4? I'm interested in using the library, but am using the newer libraries

### PR DESCRIPTION
**Pull request by mulderr**
Resolves #6.

Changed `String` to `Text` to follow a similar move in `reflex-dom`. Hope that's fine.

### CODE REVIEW

**Comment by mulderr** [comment]
Oh. Seems I forgot to update `stack.yaml`.

I'm on nixos and using reflex-platform. Kind of hoping someone actually using stack could take over this part. I have zero experience with reflex and stack.

**Comment by Wizek** [comment]
Hey! I'm looking to try out `diagrams-reflex` with my `reflex-0.5` app, and glad to see someone is already working on compatibility here. Thanks @mulderr. Is the only thing preventing this from being merged some help with `stack`? If so, I _may_ be able to help: I do have some experience with stack, although none with diagrams{.-reflex} so far.

**Comment by byorgey** [comment]
Hi @mulderr , thanks for the patch!  This looks good to me, though I don't know much about this package in particular or `reflex` in general.  I don't think updating `stack.yaml` is necessarily a blocker for merging this, although it does seem like it might be preventing our travis-ci test from completing successfully --- I am not sure how that is set up.  @bergey , any thoughts?

File: src/Graphics/Rendering/Reflex.hs [hunk 1]
```diff
@@ -232,19 +239,28 @@ renderFontWeight s = renderTextAttr "font-weight" fw
...
+  fontWeightAttr FontWeightLight      = "200"
+  fontWeightAttr FontWeightThin       = "300"
+  fontWeightAttr FontWeightNormal     = "normal"
+  fontWeightAttr FontWeightMedium     = "500"
+  fontWeightAttr FontWeightSemiBold   = "600"
+  fontWeightAttr FontWeightBold       = "bold"
+  fontWeightAttr FontWeightUltraBold  = "800"
+  fontWeightAttr FontWeightHeavy      = "900"
+  fontWeightAttr FontWeightLighter    = "lighter"
+  fontWeightAttr FontWeightBolder     = "bolder"
```
Comment (mulderr) [inline src/Graphics/Rendering/Reflex.hs:252]: I guessed the values here. Let me know if anything looks wrong.

**Comment by mulderr** [comment]
@Wizek that would be great!

**Comment by mulderr** [comment]
We seem to have a working `stack.yaml` thanks to @c-nixon.

**Comment by byorgey** [comment]
Oops, sorry I've left this for so long.  What's the status?  I assume this is still OK to merge?

**Comment by mulderr** [comment]
It should be. It's still building fine with current reflex-platform. Examples seem to be working.

**Comment by byorgey** [comment]
OK, thanks, merged!  @bergey , should we make a new release of this?

**Pull request merged**